requirements-dev.txt

```diff
@@ -1,3 +1,4 @@
+-r requirements.txt
```

Comment:
This allows `pip install -r requirements-dev.txt` also installs `requirements.txt`